invenio_vocabularies/records/jsonschemas/vocabularies/vocabulary-v1.0.0.json

```diff
@@ -5,72 +5,51 @@
   "additionalProperties": false,
   "properties": {
     "$schema": {
-      "type": "string"
+      "$ref": "local://definitions-v1.0.0.json#/$schema"
     },
-    "id": {
-      "type": "string"
-    },
-    "type": {
-      "type": "object",
-      "additionalProperties": false,
-      "properties": {
-        "id": {
-          "type": "string"
-        },
-        "pid_type": {
-          "type": "string"
-        }
-      }
-    },
-    "pid": {
-      "type": "object",
-      "additionalProperties": false,
-      "properties": {
-        "pk": {
-          "type": "integer"
-        },
-        "pid_type": {
-          "type": "string"
-        },
-        "status": {
-          "type": "string"
-        },
-        "obj_type": {
-          "type": "string"
-        }
-      }
-    },
-    "title": {
+    "description": {
       "type": "object",
-      "description": "Title of vocabulary item. Key are locale codes.",
+      "description": "Description of vocabulary item. Key are locale codes.",
       "patternProperties": {
         "^[a-z]{2}$": {
           "type": "string"
         }
       }
     },
-    "description": {
+    "icon": {
+      "$ref": "local://vocabularies/definitions-v1.0.0.json#/icon"
+    },
+    "id": {
+      "$ref": "local://definitions-v1.0.0.json#/identifier"
+    },
+    "pid": {
+      "$ref": "local://definitions-v1.0.0.json#/internal-pid"
+    },
+    "props": {
       "type": "object",
-      "description": "Description of vocabulary item. Key are locale codes.",
       "patternProperties": {
-        "^[a-z]{2}$": {
+        "^.*$": {
           "type": "string"
         }
       }
     },
```

Comment:
We probably don't have to specify the key pattern here (it looks pretty generic anyway :smile: )

Reply:
hmm I think we want to make is "tight" to localization so it should be 2 and alphabetic (not numeric)... Otherwise we can end up with pretty weird things indexed... No?

Reply:
If we agree on changing this, it can be addressed in another PR :)

Reply:
This `patternProperties` is for `props` though and that can be anything I thought.